### ISSUE TITLE
Complete POS logic

### ISSUE THREAD

**Issue by thelamer** [linuxserver/nano-wallet#26]
The pos branch has a hodge podge of non working code, but it is a starting point for shifting the logic out of Home and into Receive. 
Essentially the views will check their $route and if /pos we run some different logic outside of the standard polling in that it will stop polling once the receive amount is at or less than 0. 
* need to subtract the amount received from current amount if less than total amount regenerate new QR with new amount and update amount field
* Once amount is <= 0 stop polling in POS view and reset QR to just address
* Polling needs to happen on amount set without waiting the interval to get a current state of blocks while waiting for transactions

Basically all of this logic can be used/carried over to the standard home receive view outside of the polling start/stop if auto refresh is enabled. The QR should still reset on receives if amount set, the amount should still be deducted from, and instead of stopping the polling you just continue to display any new transactions like before.
